packages/shell-evaluator/src/shell-evaluator.ts

```diff
@@ -0,0 +1,133 @@
+import AsyncWriter from '@mongosh/async-rewriter';
+import Mapper from '@mongosh/mapper';
+import { signatures, Help } from '@mongosh/shell-api';
+
+class ShellEvaluator {
+  public mapper: any;
+  private asyncWriter: AsyncWriter;
+  private bus: any;
+  private container: any;
+  constructor(serviceProvider, bus, container?) {
+    this.mapper = new Mapper(serviceProvider, bus);
+    this.asyncWriter = new AsyncWriter(signatures);
+    this.bus = bus;
+    this.container = container;
+  }
+
+  public toReplString(): string {
```

Comment:
I think `toReplString` and `shellApiType` are not needed here. An instance of this object will never be evaluated, nor constructed inside the shell context. For what i get there is nothing that could call these two methods.